app/components/Settings/Settings.jsx

```diff
@@ -2,33 +2,68 @@ import React, { PropTypes } from 'react';
 import TextField from 'material-ui/TextField';
 import Checkbox from 'material-ui/Checkbox';
 import styles from './settings.css';
+import _ from 'lodash';
 
 class Settings extends React.Component {
     constructor(props) {
-      super(props);
-      this.setDeleteDialogPreference = this.setDeleteDialogPreference.bind(this);
+        super(props);
+
+        this.state = {
+            rootKey: window.localStorage.getItem('secretsRootKey') || '',
+            useRootKey: window.localStorage.getItem('useRootKey') || true
```

Comment:
do you think we should parse this string as bool so we don't have to  keep checking string values and just evaluate it as is?